### PR TITLE
Discord bot deployment

### PR DESCRIPTION
**Pull request by edisontim**

### CODE REVIEW

File: rust-bot/src/types.rs [hunk 1]
```diff
@@ -552,3 +547,27 @@ impl EventHandler {
...
+        env_path.push("../client/.env.production");
+
+        dotenvy::from_filename(env_path).ok();
+        match envy::from_env::<ProgramConfig>() {
+            Ok(config) => config,
+            Err(error) => panic!("{:#?}", error),
+        }
+    }
+}
+use std::path::PathBuf;
```
Comment (edisontim) [inline rust-bot/src/types.rs:573]: remove

**Comment by ponderingdemocritus** [comment]
if it runs, lgtm

File: rust-bot/Secrets.toml.example [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DISCORD_TOKEN = ""
+TORII_URL = "https://api.cartridge.gg/x/eternum-35/torii"
+NODE_URL = "https://api.cartridge.gg/x/eternum-35/katana/"
+TORII_RELAY_URL = "/ip4/127.0.0.1/udp/9091/webrtc-direct/certhash/uEiDnBV7F921DB0TUEvlf1HGFgNeNsTFUHr7snpwa1-D_xQ"
+WORLD_ADDRESS = "0x5889930b9e39f7138c9a16b4a68725066a53970d03dfda280a9e479e3d8c2ac"
```
Comment (cwastche) [inline rust-bot/Secrets.toml.example:5]: Update world address

Comment (edisontim) [reply rust-bot/Secrets.toml.example:5]: this is only an examples toml file

**Review by cwastche — APPROVED** [review]

**Pull request merged**